sdk/cosmos/azure_data_cosmos/src/clients/container_client.rs

```diff
@@ -264,19 +253,15 @@ impl ContainerClient {
         options: Option<ItemOptions<'_>>,
     ) -> azure_core::Result<Response<()>> {
         let options = options.clone().unwrap_or_default();
-        let builder = CosmosRequestBuilder::new(OperationType::Create, ResourceType::Items);
+        let builder = CosmosRequest::builder(OperationType::Create, self.items_link.clone());
         let cosmos_request = builder
```

Comment:
Non-Blocking style suggestion: I'd probably just get rid of the `builder` variable entirely and just do it all in one method chain, unless there's a need for conditionally calling certain builder functions.

There's no need to churn the PR to change this if nothing else is changing though, the code would be functionally identical in either case.